src/routes.js

```diff
@@ -1,25 +0,0 @@
-"use strict";

```

Comment:
Moved all this to `app.js` since React Router 4 doesn't require static centralized routes.